domain/api/atrium-domain-api-common/src/main/kotlin/ch/tutteli/atrium/domain/creating/IterableAssertions.kt

```diff
@@ -33,4 +33,6 @@ interface IterableAssertions {
 
     @Deprecated("Switch from Assert to Expect; will be removed with 1.0.0")
     fun <E : Any> all(plant: AssertionPlant<Iterable<E?>>, assertionCreator: (AssertionPlant<E>.() -> Unit)?): Assertion
+
+    fun hasNext(subjectProvider: SubjectProvider<Iterable<*>>): Assertion
```

Comment:
Please use `Expect<Iterable<*>>` instead of `SubjectProvider` for new functionality. SubjectProvider is a helper construct to ease the migration from `Assert` to `Expect